services/FileService.js

```diff
@@ -28,6 +28,14 @@ const saveImages = multer({
   storage: storage,
 }).array("images", MAX_IMAGE_COUNT);
 
+/**
+ * Express middleware for storing csv containing user details on local
+ * storage.
+ */
+const saveCsv = multer({
```

Comment:
What are we doing about the files once users have been created?

Reply:
It's being deleted. `unlinkSync() ` is being called. It's deleting the file.

Reply:
Okay. I guess i missed that part.